ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -78,6 +78,21 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-ipi-efa-pg-f28
+  cron: 32 15 5 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      ENABLE_OTA_TEST: OCP-23799
+    test:
+    - ref: cucushift-upgrade-healthcheck
```

Comment:
I think we do not need this health check, because the cluster did not have any change since the cluster is installed.

Reply:
I am choosing this profile is because its a new profile in e2e not not in upgrade ci:
![image](https://github.com/openshift/release/assets/83053288/ebef3e2b-6ed0-4747-ac94-75d47ffbdc35)

Reply:
it's a step in polarion, if we really do not need it, I can remove it.

Reply:
Install workflow already did a round of health check before run `test` part.